app/models/course_year.rb

```diff
@@ -59,8 +59,8 @@ def compute_user_course_module_progress(lessons_with_progresses, modules_in_orde
   end
 
   def course_module_progress_status(lessons)
```

Comment:
not for this PR but we should move this to a service class later. `CourseYear` does not need to know anything about the progress. Also this method despite being in this class never actually references the `CourseYear` instance